src/Test/Lint.hs

```diff
@@ -94,11 +94,11 @@ main = shaken test $ \args obj -> do
         need [obj "tracker-source.c", obj "tracker-source.h"]
         cmd "gcc" ["-c", obj "tracker-source.c", "-o", out]
     where gen t f = unit $ if isWindows
-                           then cmd "cmd /c" ["echo" ++ t ++ " > " ++ f]
-                           else cmd Shell "echo" ["x", ">", f]
+                           then cmd "cmd /c" ["echo " ++ t ++ " >" ++ f]
+                           else cmd "sh -c" ["echo " ++ t ++ " >" ++ f]
```

Comment:
Doesn't `Shell` do roughly `sh -c`? Although I guess they should either both use `Shell` for the test, or neither use it (and here, I think actually they're sufficiently similar that `cmd Shell ["echo " ++ t ++ " > " ++ f]` might cover both).